src/SourceBuild/Arcade/tools/TextOnlyPackages.csproj

```diff
@@ -0,0 +1,68 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFrameworks>net6.0</TargetFrameworks>
+    <RestorePackagesPath>$(TargetPackagesPath)</RestorePackagesPath>
+  </PropertyGroup>
+
+  <!--
+    The following text-only packages are ones that are consumed by the installer repo, whose
+    versions are dynamic.  These are maintained here so the dynamic version numbers can
+    come from the source (eng/Versions.props).  All other more static text-only packages
+    should be added to source-build-reference-packages.
+  -->
+  <ItemGroup>
+    <PackageDownload Include="Microsoft.DotNet.Common.ItemTemplates"
+                     Version="[$(MicrosoftDotNetCommonItemTemplates21PackageVersion)];
+                              [$(MicrosoftDotNetCommonItemTemplates30PackageVersion)];
+                              [$(MicrosoftDotNetCommonItemTemplates31PackageVersion)];
+                              [$(MicrosoftDotNetCommonItemTemplates50PackageVersion)];
+                              [$(MicrosoftDotNetCommonItemTemplates60PackageVersion)]" />
+
+    <PackageDownload Include="Microsoft.DotNet.Web.ItemTemplates"
```

Comment:
@MichaelSimons it looks like this is failing, did I merge the source build content correctly?